src/feat/wave-reader.cc

```diff
@@ -354,6 +361,10 @@ void WaveData::Write(std::ostream &os) const {
   }
   if (os.fail())
     KALDI_ERR << "Error writing wave data to stream.";
+  if (num_clipped > 0)
+    KALDI_WARN << "WARNING: clipped " << num_clipped
+               << " samples out of total " << num_chan * num_samp
+               << ". Reduce volume.";
 }
```

Comment:
Could you please change the punctuation from "Reduce volume." to "Reduce volume?".  Just to make it more of a suggestion than an imperative.